src/api/product/services/getMany.js

```diff
@@ -6,9 +6,34 @@ async function readMany(req, res) {
   try {
     const limit = +req.query.limit;
     const pageNo = +req.query.pageNo;
+    const priceTo = +req.query.priceTo;
+    const priceFrom = +req.query.priceFrom;
+    const { language } = req.headers;
+    const { search } = req.query;
+    const { notSelectedLanguages } = req;
     const filter = { deleted: false };
 
-    return sendPaginatedList(res, Product, filter, pageNo, limit);
+    const dispatchedLanguage = {
+      [`name${req.headers.language}`]: 'name',
+      [`description${req.headers.language}`]: 'description',
+    };
+
+    const select = `-name${notSelectedLanguages[0]} -name${notSelectedLanguages[1]} -description${notSelectedLanguages[0]} -description${notSelectedLanguages[1]}`;
+    filter['$or'] = [
```

Comment:
Add if `search` exist check, `search` is not required for our api call, it may not be there